### DISCUSSION TITLE
Nom de la commande /echangefamilier non alignée avec les autres

### DISCUSSION THREAD

**Discussion by cdamiens** [Crownicles/Crownicles#1562] [Nouvelles idées]
Bonjour,
J'ai remarqué que la commande /echangefamilier est une des rares commandes à ne pas commencer par un verbe à l'infinitif.
Toutes les autres commandes relatives aux familiers (et une grande majorité des autres commandes) sont de la forme:
- transfererfamilier
- libererfamilier
- vendrefamilier

Y a-t-il une bonne raison pour que /echangefamilier ne suive pas ce format?

**Comment by BastLast** [comment]
Bonjour, c'est une erreur, merci de poster un commentaire dans l'issue dédiée aux typos 👌

Bien vu !

**Reply by cdamiens** [reply]
Ok, je vais l'ajouter à l'issue des typo.
